elastic/kibana
# Add support for geo_shape fields as the entity geospatial field when creating tracking containment alerts

Closes https://github.com/elastic/kibana/issues/163996

### To test
1) Checkout [fake tracks geo_shape branch](https://github.com/nreese/faketracks/tree/geo_shape)
2) run npm install
3) run `node ./generate_tracks.js`
4) in kibana, create `tracks*` data view
5) create map, use "create index" and draw boundaries that intersect tracks. See screen shot
    <img width="500" alt="Screen Shot 2023-08-17 at 2 49 52 PM" src="https://github.com/elastic/kibana/assets/373691/5f1444d7-2e12-4dd2-99c1-c730c2157e04">
6) create geo containment alert where entity index is `tracks*` and boundaries index is `boundaries`.
7) Verify alerts get generated with entity geo_shape locations
